themes/app/src/js/components/CustomNav.vue

```diff
@@ -1,18 +1,18 @@
 <template>
-    <nav class="custom-nav">
-        <div class="top-menu">
-            <a href="/"><img id="logo"/></a>
+    <nav class="customnav">
```

Comment:
ah I thought you got what I meant when I said update to `main-nav` instead

I also meant update all the prefix classes throughout the pull request :) So at the end of the refactor we have no instances of `customnav` and they are all replaced with `main-nav` :)

Reply:
ok, will do now